app/tasks/maintenance_tasks/task.rb

```diff
@@ -1,11 +1,19 @@
 # frozen_string_literal: true
 
+require 'maintenance_tasks/active_record_task'
+require 'maintenance_tasks/array_task'
+require 'maintenance_tasks/csv_task'
+# TODO: require other adapters, or make this dynamic somehow
+
```

Comment:
I think `app/tasks` would make more sense _if we're planning on keeping it around_ (otherwise, `app/models` makes sense). I think I'd seen some discussion about not using `app/tasks` internally at all, but maybe I'm wrong.